service/health/searcher/aws.go

```diff
@@ -2,12 +2,43 @@ package searcher
 
 import (
 	"context"
+
+	"github.com/giantswarm/health-service/service/health/key"
```

Comment:
Nitpick:
We usually sort into 3 blocks 
1. standard libs like `context`
2. external libs
3. project internal libs like this this `key` package

You can find info like this in `fmt`: https://github.com/giantswarm/fmt/blob/master/go/imports.md